### ISSUE TITLE
Add @qm_1v1 support in !rank

### ISSUE THREAD

**Issue by MazGrasl** [Taloth/Aoe4bot#5]
Example: "!rank DeMuslim @qm_1v1" should display DeMuslims unranked 1v1 stats

**Comment by Taloth** [comment]
Do you think there's an usecase here? it was just a thought. For beasty's channel we have !rank !rank2v2 !rank3v3 commands with the leaderboard query param.

If we do it then we should support @qm_*, @rm_*. But also @1v1, @2v2, @3v3, @4v4 defaulting to ranked if it exists. I assume we'll get team ranked eventually.

**Comment by MazGrasl** [comment]
I'd consider this lowest priority - it would free up those separate commands, and it's not too hard to add.
I agree that it would be best to include a wide variety of combinations - parsing those is probably the toughest part.

Once this is done it might be nice to add it to the other commands as well.

**Comment by Taloth** [comment]
Yes, absolutely low prio.

> Once this is done it might be nice to add it to the other commands as well.

My first thought was: For what purpose?
!match doesn't need it, unless you want to filter `games` and return the last match in a particular ladder.
!winrate same deal, unless you want to see the winrate on a particular ladder.
So yeah, adding it to other commands would mean filtering results.

**Comment by MazGrasl** [comment]
Yes that was my thought, for !match to get the last 4v4 match for example.
For !winrate right now it includes all games across different modes, which is a bit weird. Having a command that only includes games for certain game modes could be beneficial.
